docker-compose/monitor/README.md

```diff
@@ -126,7 +126,7 @@ to emit traces to the OpenTelemetry Collector which, in turn, will aggregate the
 
 Start the local stack needed for SPM, if not already done:
 ```shell
-docker compose up
+docker compose -f docker-compose-v2.yml up
```

Comment:
The v2 files need to be renamed just docker-compose, so that the user is not required to specify them on the command line.